reactor-core/src/test/java/reactor/core/QueueSubscriptionTest.java

```diff
@@ -81,9 +81,9 @@ public String toString() {
 				})
 		.withMessage("\n" +
 			"Expecting actual:\n" +
-			"  \"foo\"\n" +
-			"and actual:\n" +
 			"  ThisIsNotAQueue\n" +
+			"and:\n" +
+			"  \"foo\"\n" +
```

Comment:
I don't think that this change belongs to this PR. Wdyt?

Reply:
the assertion message in assertj was faulty (reversing the _actual_ and _expected_, and referring to an _actual_ twice). it has been fixed in the assertj release that is upgraded in the PR (see https://togithub.com/assertj/assertj-core/issues/2565)